runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -636,7 +636,9 @@ class INTERPRETER_CLASS
 		void* const jitReturnAddress = VM_JITInterface::fetchJITReturnAddress(_currentThread, _sp);
 		J9ROMMethod* const romMethod = J9_ROM_METHOD_FROM_RAM_METHOD(_sendMethod);
 		void* const exitPoint = j2iReturnPoint(J9ROMMETHOD_SIGNATURE(romMethod));
-		if (J9_ARE_ANY_BITS_SET(romMethod->modifiers, J9AccNative | J9AccAbstract)) {
+
+		bool isDefCon = (J9_BCLOOP_SEND_TARGET_DEFAULT_CONFLICT ==  J9_BCLOOP_DECODE_SEND_TARGET(_sendMethod->methodRunAddress));
+		if (isDefCon || J9_ARE_ANY_BITS_SET(romMethod->modifiers, J9AccNative | J9AccAbstract)) {
```

Comment:
The local variable is not needed since it is only used in one place.
```suggestion
		if (J9_ARE_ANY_BITS_SET(romMethod->modifiers, J9AccNative | J9AccAbstract)
		|| (J9_BCLOOP_SEND_TARGET_DEFAULT_CONFLICT ==  J9_BCLOOP_DECODE_SEND_TARGET(_sendMethod->methodRunAddress)
		) {
```